api/discovery_chain.go

```diff
@@ -109,9 +109,17 @@ type CompiledDiscoveryChain struct {
 	// non-customized versions.
 	CustomizationHash string
 
+	// Default indicates if this discovery chain is based on no
```

Comment:
Should the API docs be updated to account for these changes.

Reply:
IIRC we don't currently document the full disco chain response at all.